packages/stylelint-config-zone/README.md

```diff
@@ -30,5 +24,3 @@ Add the Zone config to your stylelint config. Be sure to add as the last item.
   // ... project config here, if any ...
 }
 ```
-
-**Optional but recommended:** Remove duplicated rules from your stylelint config.
```

Comment:
❓ Why do we want to remove this?

Reply:
Good question! It doesn't really make any difference if you remove duplicates or not.

Reply:
True, I just thought that in general it is nicer to have more order in the configs 😄